exercises/circular-buffer/circular_buffer_test.py

```diff
@@ -7,40 +7,37 @@
 )
 
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.1
 class CircularBufferTest(unittest.TestCase):
     def test_read_empty_buffer(self):
         buf = CircularBuffer(1)
         with self.assertRaises(BufferEmptyException):
             buf.read()
 
+    def test_read_just_written_item(self):
+        buf = CircularBuffer(1)
+        buf.write('1')
+        self.assertEqual('1', buf.read())
```

Comment:
By convention, we use `assertEqual(actual, expected)` for our tests - can you switch the argument order in all of these tests?

You can check the rest of our conventions in the main [README.md](https://github.com/exercism/python/blob/master/README.md).

Reply:
Changed then. Thanks to point it out